public class ArithmeticExceptionExample {
    public static void main(String[] args) {
        int dividend = 10;
        int divisor = 0;
//        try {
//            int result = dividend / divisor;
//            System.out.println("Result: " + result);
//        } catch (ArithmeticException e) {
//            System.out.println("ArithmeticException occurred: " + e.getMessage());
//        } finally {
//            System.out.println("This is the example of the ArithmeticException");
//        }

        int result = dividend / divisor;
    }
}